src/tests/mod.rs

```diff
@@ -1901,3 +1903,46 @@ macro_rules! generate_ops_tests {
         }
     };
 }
+
+#[test]
+fn fraction_from_float() {
```

Comment:
The `src/tests/mod.rs` is a place for somewhat _generic_ typeless math and op tests.
I guess this `fraction_from_float` would be better placed in `src/fraction/mod.rs`, because it's a GenericFraction test?
Should we move it there?